### ISSUE TITLE
Chapter 2 - 2.3.1 - Incorrect figure number

### ISSUE THREAD

**Issue by amanjeev** [apollolabsdev/wiredworldbook#1]
2. Interfacing GPIO Peripherals
2.3.1 Driving an external load/device through a microcontroller output pin

Quote where 

> Note that between the two configurations in Figure 11 the direction of current differs. On the left hand side the current comes out from the pin towards ground. However, on the right hand si....

Should this be Figure 13?

**Comment by apollolabsdev** [comment]
That is correct! It should be 13, not 11. I will correct the typo in an updated version of the book. I found a grammatical mistake in the same section too :)
